pyproject.toml

```
@@ -16,7 +16,7 @@
 # under the License.
 [tool.poetry]
 name = "pyiceberg"
-version = "0.7.1"
+version = "0.7.2"
```

We will update this version when and if we release 0.7.2
```suggestion
version = "0.7.1"
```